.github/workflows/release-apidocs.yml

```diff
@@ -0,0 +1,140 @@
+on:
+  push:
+    branches:
+      - mainnet
+      - testnet
+      - integrationnet
+      - develop
+  pull_request:
+
+name: Publish API docs
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+  AWS_REGION:
+    "us-west-1"
+
+  #        if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+jobs:
+  terraform:
+    name: Terraform
+    environment:
+      name: ${{ (github.ref == 'refs/heads/mainnet' && 'mainnet') || (github.ref == 'refs/heads/integrationnnet' && 'integrationnet') || (github.ref == 'refs/heads/testnet' && 'testnet') || 'dev' }}
```

Comment:
we do have different branches here. `release/mainnet`, `release/integrationnet` and `release/testnet`